man/systemd.exec.xml

```diff
@@ -930,6 +930,19 @@ CapabilityBoundingSet=~CAP_B CAP_C</programlisting>
         <varname>SystemCallFilter=~@mount</varname>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>TemporaryFileSystem=</varname></term>
+
+        <listitem><para>Takes a space-separated list of mount points for temporary file system (tmpfs). If sets, new
+        file system namespace for executed processes is set up. Each mount point may be prefixed with <literal>-</literal>,
+        in which case it will be ignored when it does not exist. Each mount point may optionally be suffixed with a colon
+        (<literal>:</literal>) and mount option such as <literal>size=10%</literal> or <literal>ro</literal>. If no
+        mount option is specified, then the tmpfs is mounted with <literal>mode=0755</literal>.
+        Note that each tmpfs enables <literal>nodev</literal> and <literal>strictatime</literal> options, that is,
+        each tmpfs is mounted with <literal>MS_NODEV|MS_STRICTATIME</literal> mount flags.
```

Comment:
I think this sentence needs to say "by default" or similar.

AFAICT, from looking at the implementation commit and the test for mount_option_mangle(), it's possible to pass ":nodev,noatime" and override the default options.